src/code.cloudfoundry.org/gorouter/accesslog/file_and_loggregator_access_logger.go

```diff
@@ -12,7 +12,7 @@ import (
 	log "code.cloudfoundry.org/gorouter/logger"
 )
 
-//go:generate counterfeiter -o fakes/accesslogger.go . AccessLogger
+//counterfeiter:generate -o fakes/accesslogger.go . AccessLogger
```

Comment:
I find it a bit confusing that some need both directives and others only one but I get that we only need to run this once per package. Should we maybe move the `//go:generate` to a `gen.go` file then?